test/verify/machineslib.py

```diff
@@ -1878,6 +1894,12 @@ def createAndVerifyVirtInstallArgs(self):
             wait(lambda: self.machine.execute(virt_install_cmd), delay=3)
             virt_install_cmd_out = self.machine.execute(virt_install_cmd)
             self.assertTrue("--install os={}".format(self.os_short_id in virt_install_cmd_out))
+            if self.is_unattended:
+                self.assertTrue("profile={0}".format('desktop' if self.profile == 'Workstation' else 'jeos') in virt_install_cmd_out)
```

Comment:
Please use `assertIn(needle, haystack)`, that provides much more useful output on failures.

Reply:
Fixed.